src/styles/core/colors/_light.scss

```diff
@@ -199,11 +199,11 @@
   --color-card-shadow: #{change-color(dark-color(fill), $alpha: 0.04)};
   --color-link-block-card-border: #{change-color(dark-color(fill), $alpha: 0.04)};
 
-  --color-standard-red: #8B0000;
-  --color-standard-orange: #8B4000;
-  --color-standard-yellow: #8F7200;
-  --color-standard-blue: #002D75;
-  --color-standard-green: #023B2D;
-  --color-standard-purple: #512B55;
-  --color-standard-gray: #{dark-color(fill-tertiary)};
+  --color-standard-red: #FFC2C2;
```

Comment:
I'm slightly concerned about the contrast of the secondary gray text against these new lighter colors.

There are a couple of places where we use the gray text with varying sizes that appear in the foreground against these standard colors:
* eyebrow text
* badge text
* IDE language switcher text

The majority of these new colors only have a sufficient contrast ratio for large text, like the eyebrow—and even the eyebrow was a little bit hard for me to distinguish personally.

For reference, [this][hig-contrast] is the relevant section of the HIG on sufficient contrast ratios for ranges of text sizes. And here are the ratios for the gray text against these light colors:
* [blue][contrast-blue] (3.4:1)
* [green][contrast-green] (3.24:1)
* [orange][contrast-orange] (3.92:1)
* [purple][contrast-purple] (3.73:1)
* [red][contrast-red] (3.75:1)
* [yellow][contrast-yellow] (4.49:1)

We may want to consider one of the following things to try and improve this so that we don't have a noticeable regression in the contrast ratios:
* choose lighter values for these new light mode variants of the standard colors
* use a darker secondary gray color for the problematic text
* use a heavier font weight for the secondary gray text or only the small sized secondary gray text

[hig-contrast]: https://developer.apple.com/design/human-interface-guidelines/accessibility#Color-and-effects
[contrast-blue]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=99CCFF
[contrast-green]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=99CCCC
[contrast-orange]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=FFCC99
[contrast-purple]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=CCCCFF
[contrast-red]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=FFC2C2
[contrast-yellow]: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=FFE0A3

Reply:
Ended up using a darker color for the text.